test/logic/daily_digest_test.rb

```diff
@@ -1,11 +1,37 @@
 require 'test_helper'
 
 class DailyDigestTest < ActionMailer::TestCase
+  fixtures :lists
+  fixtures :items
+
+  def setup
+    @digest = DailyDigest.new
+  end
+
+  test "should return statistic" do
+    list = lists(:first)
+
+    digest.expects(:completed_for_today).returns(list.items.completed)
```

Comment:
@vlasiak So here you expect that `completed_for_today` is being called and you mock its results, right? I'm wondering can you instead mock `Item.completed_today` call (in order to do not break encapsulation)?